web_src/js/utils/dom.js

```diff
@@ -51,8 +51,22 @@ export function isElemHidden(el) {
   return res[0];
 }
 
-export function queryElemSiblings(el, selector = '*') {
-  return Array.from(el.parentNode.children).filter((child) => child !== el && child.matches(selector));
+function applyElemsCallback(elems, fn) {
+  if (fn) {
+    for (const el of elems) {
+      fn.call(el, el); // call the function with the element as "this", and pass the element as the first argument
+    }
```

Comment:
I wouldn't encourage `this` usage for new code, so this should just be `fn(el)`.

Reply:
See https://github.com/go-gitea/gitea/pull/30191#pullrequestreview-1970253726